tests/test_config_reader.py

```diff
@@ -0,0 +1,12 @@
+import yaml
+import os
+
+from pytentiostat.config_reader import parse_config_file, get_adv_params
+
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+
+def test_parse_config_files():
+    confdir = os.path.join(THIS_DIR, 'static/')
+    config_data = parse_config_file(confdir)
+    dict_check = isinstance(config_data, dict)
+    assert dict_check == True
```

Comment:
you can just write this as 
```
assert isinstance(config_data, dict)
```
and delete hte previous line.  It will be easier to read (and less to maintain)!

Reply:
ah, okay implemented!